src/components/BarChart/BarChart.props.ts

```diff
@@ -0,0 +1,19 @@
+export interface IBarChartProps {
+    id: string;
+    barColor?: string;
+    hovColor?: string;
+    data: Array<IBarChartData>;
+    tipText?: (data: IBarChartData) => string;
+    xAxisFormat?: () => string;
+    width?: number;
+    height?: number;

```

Comment:
Component scales based on width and aspect ratio so there is no need of min/max props for height